binary_port/src/error_code.rs

```diff
@@ -186,6 +186,12 @@ pub enum ErrorCode {
     /// the case that nothing else works.
     #[error("The transaction or deploy sent to the network was invalid for an unspecified reason")]
     InvalidTransactionOrDeployUnspecified = 57,
+    #[error("the switch block for the requested era was not found")]
+    SwitchBlockNotFound = 58,
```

Comment:
Can you please provide doc comments for the new variants? I know they are somehow superfluous here, but just for the consistency-sake.

We might also consider removing all doc comments.

Reply:
done